cmd/gctcli/commands.go

```diff
@@ -2236,6 +2236,39 @@ var cancelAllOrdersCommand = &cli.Command{
 	},
 }
 
+var modifyOrderCommand = &cli.Command{
+	Name:      "modifyorder",
+	Usage:     "modify price and/or amount of a previously submitted order",
+	ArgsUsage: "<exchange> <asset> <pair> <order_id>",
+	Action:    modifyOrder,
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "exchange",
+			Usage: "exchange this order is submitted to",
+		},
+		&cli.StringFlag{
+			Name:  "asset",
+			Usage: "required asset type",
+		},
+		&cli.StringFlag{
+			Name:  "pair",
+			Usage: "required trading pair",
+		},
```

Comment:
You might be able to add:
```go
Required: true,
```

Reply:
Truth is user can modify just one of them. If required==true, both will have to be supplied, which would be an artificial requirement.